debian/rules

```diff
@@ -3,3 +3,5 @@
 export PYBUILD_NAME=mu
 %:
 	dh $@ --with python3 --buildsystem=pybuild
+
+override_dh_auto_test:
```

Comment:
This might be a stupid question, but woudn't the colon from this line indicate that perhaps there should be a command after it?

Reply:
It's not required. It's just a makefile and we're telling that target to do nothing. We could add a 'true' there to make it more readable, but it behaves the same way.
I didn't add anything because this is the way the example is given in the dh manpage.